src/main/flight/gps_rescue.c

```diff
@@ -303,13 +298,18 @@ void idleTasks()
         return;
     }
 
+    // Don't update any rescue flight statistics if we haven't applied a proper altitude offset yet
+    if (!isAltitudeOffset()) {
```

Comment:
There is a lot of change in here now that seems to be unrelated to the intent of the pull request - could it be that you have rebased this on an older version of master?

Reply:
By all means, the change right here is not related to a variable name change, and should not be done at this stage.
Also, besides the fact that #6329 has already been released, there is absolutely no reason to revert a pull request. The correct way to resolve conflicts is to resolve them in the pull request to be merged, not by reverting past pull requests.